components/JumboSection.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <section class="hero is-large">
+    <div class="hero-body" :style="{ backgroundImage: 'url(' + image + ')' }">
+      <div class="container">
+        <h1 class="title" v-html="title">
+          {{ title }}
+        </h1>
+      </div>
+    </div>
+    <!-- <div
```

Comment:
I propose that commented code would be deleted for the clarity of the code. (from line #10 ~ #22)